addon/models/user.js

```diff
@@ -2,6 +2,7 @@ import Ember from 'ember';
 import DS from 'ember-data';
 import config from 'ember-get-config';
 
+import { authenticatedAJAX } from 'ember-osf/utils/ajax-helpers';
```

Comment:
What is `ember-osf/utils/ajax-helpers`? You should be able to use the `authenticatedAJAX()` method from the `current-user` service.